### DISCUSSION TITLE
Is it possible to use "limit" on nav / collection without pagination?

### DISCUSSION THREAD

**Discussion by niloc7** [statamic/cms#4810] [Q&A] [answered]
After experimenting a hundred ways... I think that "limit" can only be used in pagination or metadata..

I have a collection, ie. "products", and I want to get the first 5 to become links, and then next 5 to be used somewhere else.. just more customized navigation, instead of having to be stuck with the default pagination..

But limit doesn't work, and there doesn't seem to be a "from x to x" to select first to fifth slug etc..

Hmm.. am I missing something here or is this just how Statamic works for now?

Thanks!

**Comment by niloc7** [comment]
OMG I THINK I KNOW WHAT WENT WRONG

I've always been using collection:products | limit:5

But all the while.. it's totally different, it should have been collection:products limit="5"

Hmph.. weirdly it works when in that format..

**Comment by jackmcdade** [comment, marked as answer]
Combine limit=“5” with offset=“5” to get the _next_ chunk, and so on.
